extensions/wikia/Chat2/js/controllers/controllers.js

```diff
@@ -91,6 +103,9 @@ var NodeChatSocketWrapper = $.createClass(Observable, {
 	},
 
 	onMsgReceived: function (message) {
+		this.log('Message received: ' + message.event);
+		this.log(message);
+
 		switch (message.event) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a "default" clause to this "switch" statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ASwitchWithoutDefault)